docs/img/readme/data_model.py

```diff
@@ -0,0 +1,115 @@
+import json
+
+import numpy as np
+import pandas as pd
+
+if __name__ == '__main__':
+  # Mapping Flatland 3 to generic data model if only one submission
+  # - scenario run ID -> submission_id
+  # TODO generalize to multiple submissions aka. scenario runs?
+  # - metric          -> (submission_id,test_id,env_id, "normalized_reward")
+  # - kpi             -> (submission_id, test_id, "normalized_reward_test")
+  # - objective       -> (submission_id, "normalized_reward_all_tests")
+
+  # =======================================================
+  # Scenario Run Evaluation
+  # =======================================================
+  # filename,Unnamed: 0,test_id,env_id,n_agents,x_dim,y_dim,n_cities,max_rail_pairs_in_city,n_envs_run,seed,grid_mode,max_rails_between_cities,malfunction_duration_min,malfunction_duration_max,malfunction_interval,speed_ratios,reward,normalized_reward,percentage_complete,steps,simulation_time,nb_malfunctioning_trains,nb_deadlocked_trains,controller_inference_time_min,controller_inference_time_mean,controller_inference_time_max
+  stats = json.load(open("sub-37557fc0-222e-4398-81d6-eade2892c9f6.json"))
+  df = pd.read_csv("sub-37557fc0-222e-4398-81d6-eade2892c9f6.csv")
+  print(stats)
+  # {'state': 'FINISHED', 'progress': 1.0, 'simulation_count': 50, 'total_simulation_count': 150, 'score': {'score': 41.62435556467708, 'score_secondary': 0.597},
+  #  'meta': {'normalized_reward': 0.83249,
+  #           'termination_cause': 'The mean percentage of done agents during the last Test (10 environments) was too low: 0.176 < 0.25',
+  #           'private_metadata_s3_key': 'results/sub-37557fc0-222e-4398-81d6-eade2892c9f6.csv', 'reward': -3859.26, 'percentage_complete': 0.597}}
+
+  # score == normalized_reward.sum()
+  sum_normalized_reward = df["normalized_reward"].sum()
```

Comment:
Beware: 
pandas has [DataFram.sum](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.sum.html) has one method with option `skipnabool, default True`
whereas numpy has two different methods [nansum](https://numpy.org/doc/stable/reference/generated/numpy.nansum.html) and [sum](https://numpy.org/doc/stable/reference/generated/numpy.sum.html)